### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by puzak-cws**

### CODE REVIEW

File: android/ta_player_plugin/src/main/java/com/applicaster/plugin/televisionacademyplayer/PlayerContract.kt [hunk 1]
```diff
@@ -135,7 +139,12 @@ class PlayerContract : BasePlayer(), ApplicationLoaderHookUpI {
     }
 
     override fun executeOnApplicationReady(context: Context?, listener: HookListener?) {
-        BitmovinCastManager.initialize()
+        Log.d("TAG", "executeOnApplicationReady")
+        BitmovinCastManager.getInstance().addMessageReceivedCallback { var1, var2, var3 ->
+            Log.d("TAG", "addMessageReceivedCallback var1 : $var1 var2 : $var2 var3 % $var3 ")
+        }
+        BitmovinCastManager.initialize("F02DA75A", "some_meessage_namespace", ExpandedControlsActivity::class.java)
+        BitmovinCastManager.getInstance().updateContext(context)
```
Comment (VladSumtsov) [inline android/ta_player_plugin/src/main/java/com/applicaster/plugin/televisionacademyplayer/PlayerContract.kt]: make application ID configurable via zapp please.

File: android/ta_player_plugin/build.gradle [hunk 2]
```diff
@@ -64,6 +68,11 @@ dependencies {
             exclude group: 'com.google.android.exoplayer', module: 'exoplayer-smoothstreaming'
         }
     }
+//    implementation project(':cast_plugin')
+    implementation("com.applicaster:GenericChromecast-Android-no-meta:1.0.+") {
```
Comment (VladSumtsov) [inline android/ta_player_plugin/build.gradle]: Why do we need a dependency on the plugin here?

File: android/ta_player_plugin/src/main/java/com/applicaster/plugin/televisionacademyplayer/PlayerContract.kt (hunk 1 above)
Comment (puzak-cws) [reply android/ta_player_plugin/src/main/java/com/applicaster/plugin/televisionacademyplayer/PlayerContract.kt]: It was just for debugging purposes, no longer needed. Thanks.

File: android/ta_player_plugin/build.gradle (hunk 2 above)
Comment (puzak-cws) [reply android/ta_player_plugin/build.gradle]: Yep, you're right.

**Pull request merged**